extension/src/experiments/columns/model.test.ts

```diff
@@ -10,21 +10,32 @@ import {
   deeplyNestedOutput,
   columns as deeplyNestedColumns
 } from '../../test/fixtures/expShow/deeplyNested'
+import {
+  dataTypesOutput,
+  columns as dataTypesColumns
+} from '../../test/fixtures/expShow/dataTypes'
 
 describe('ColumnsModel', () => {
   const exampleDvcRoot = 'test'
 
-  it('should return columns that equal the columns fixture when given the output fixture', async () => {
+  it('should return expected columns when given the default output fixture', async () => {
```

Comment:
Made all the output tests for row and column outputs of the fixtures to use similar titles.

Reply:
Yeah, that reads a bit more naturally